sabnzbd/newswrapper.py

```diff
@@ -219,7 +219,7 @@ def __init__(self, host, port, info, sslenabled, send_group, nw, user=None, pass
             if(sabnzbd.cfg.enable_nntps_verification() == 0):
                 ctx.verify_mode = ssl.CERT_NONE
 
-            self.sock = ctx.wrap_socket(socket.socket(af, socktype, proto), server_hostname=host)
+            self.sock = ctx.wrap_socket(socket.socket(af, socktype, proto), server_hostname=nw.server.host)
         elif sslenabled and not HAVE_SSL:
             logging.error(T('Error importing OpenSSL module. Connecting with NON-SSL'))
```

Comment:
I know, but the text (and thus it's translations) have been there for a long time. 
Although it might be worth a change..